docs/guides/build-dind.md

```diff
@@ -0,0 +1,166 @@
+> New to KubeCI engine? Please start [here](/docs/concepts/README.md).
+
+# Build From Source
+
+This tutorial will show you how to build and push container image from a Github source repository using [dind](https://github.com/GoogleContainerTools/dind).
+
+Before we start, you need to have a Kubernetes cluster, and the kubectl command-line tool must be configured to communicate with your cluster. If you do not already have a cluster, you can create one by using [Minikube](https://github.com/kubernetes/minikube). Now, install KubeCI engine in your cluster following the steps [here](/docs/setup/install.md).
+
+## Create Secret
+
+In order to push image into container registry, you have to configure credential-initializer. First, create a secret with docker credentials and proper annotations. Also, you need to specify this secret in workflow's service-account in order to configure credential-initializer.
```

Comment:
`Also, you need to specify this secret in workflow's service-account in order to configure credential-initializer.`

What does it mean?